Project/src/main/java/app/models/Role.java

```diff
@@ -1,20 +1,19 @@
 package app.models;
 
```

Comment:
Aside from agreeing this should be an enum,

Since you changed something (in a major way) that already exists, this should have been its own PR